package.json

```diff
@@ -40,10 +40,11 @@
     "gulp-mocha": "2.2.0",
     "mocha": "2.3.4",
     "s": "0.1.1",
-    "superagent": "0.15.4"
+    "superagent": "0.15.4",
+    "uws": "^0.3.3"
```

Comment:
Well, since it's in the devDependencies, and all tests pass, you won't have to PR that much. Only when some other update really breaks a test.